matson.libraries.yml

```diff
@@ -103,6 +104,8 @@ matson:
   js:
     js/vendor/modernizr.js: {}
     js/build/scripts.js: {}
+    js/build/sidebar.js: {}
+    node_modules/scrolltofixed/jquery-scrolltofixed-min.js: {}
```

Comment:
I was trying to avoid that but I guess we have no choice here.  I'll move the js to the vendor folder and add it to git.